### DISCUSSION TITLE
starlette`s new version has a fix in 0.37.1, need to update to it.

### DISCUSSION THREAD

**Discussion by aswanidutt87** [fastapi/fastapi#11172] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [ ] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [ ] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [ ] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
please update the starlette dependency to 0.37.1
project.toml file
dependencies = [
    "starlette>=0.36.3,<0.37.0",  to 
dependencies = [
    "starlette>=0.36.3,<=0.37.1"
```

### Description

on February 9, 2024
 In 0.37.1 of Starlette the issue of " Warn instead of raise for missing env file on Config " https://github.com/encode/starlette/pull/2485. has been addressed and fastapi needs to update its dependency of starlette to this version to avoid the raising of exception in case of missing env file on config

### Operating System

macOS

### FastAPI Version

0.109.2

### Pydantic Version

Version: 2.16.2

### Python Version

3.9

**Comment by aswanidutt87** [comment]
this keeps causing the app throw a "FileNotFoundError: Config file '.env' not found." from Starlette`s old version

**Comment by Kludex** [comment, marked as answer]
It will be updated soon.

**Reply by aswanidutt87** [reply]
@Kludex the latest version is still having old starlette`s version. here is the PR to upgrade this dependency.https://github.com/tiangolo/fastapi/pull/11266 , please approve and merge .    "starlette>=0.37.2,<0.38.0",   
 Thank you
